test_runner/regress/test_compatibility.py

```diff
@@ -552,3 +578,29 @@ def test_historic_storage_formats(
     env.pageserver.http_client().timeline_compact(
         dataset.tenant_id, existing_timeline_id, force_image_layer_creation=True
     )
+
+
+@check_ondisk_data_compatibility_if_enabled
+@pytest.mark.xdist_group("compatibility")
+@pytest.mark.parametrize("combination", all_pairs_component_versions(), ids=fixtures.utils.comb_ids)
```

Comment:
Ideally, It would be nice to build-in all `parameterize`parameters into one function:
I.e. instead of 
```python
@pytest.mark.parametrize("combination", all_pairs_component_versions(), ids=fixtures.utils.comb_ids)
``` 

make it 
```python
@pytest.mark.parametrize(all_pairs_component_versions())
# or, probably it'll be 
@pytest.mark.parametrize(**all_pairs_component_versions())
```

Reply:
done